### ISSUE TITLE
How to test on other dataset?

### ISSUE THREAD

**Issue by Yu-fei-Han** [sony/NeISF#2]
Thank you for your work!

I was wondering if you have tested your model on the PANDORA dataset. I have attempted this multiple times, but I only achieve rendered images and poor mesh reconstruction. Could you please provide me with some guidance or assistance?

I am looking forward to your response!

**Comment by ligoudaner377** [comment]
Hi @Yu-fei-Han 
Thank you for your interest in this work.
Though not officially reported in the paper, we have tested PANDORA dataset on our model.
Here are some surface normal reconstruction results:
<img width="142" alt="Picture2" src="https://github.com/user-attachments/assets/43b3d15d-af5e-423f-a58e-9cf1fc5aec11">
<img width="142" alt="Picture1" src="https://github.com/user-attachments/assets/eff07bc4-1c92-46cc-bccb-80355171e8f1">

For testing the customized data, the camera poses might be the problem.
We suggest visualizing the camera poses using the provided [script.](https://github.com/sony/NeISF/blob/main/visualize_cameras.py) 

Also, the intensity of images in PANDORA is so low that the model may fail to learn the geometries.
We have multiplied the intensity with a gain before training the model:
s0 = s0 * gain
s1 = s1 * gain
s2 = s2 * gain
(gain was usually set to 3.0-6.0)

feel free to let me know if you have further questions
